### PR TITLE
update (doc/awscloudformation) added note for user

### PR DESCRIPTION
**Pull request by shashiranjan84**

### CODE REVIEW

File: app/install/aws-cloudformation.md [hunk 1]
```diff
@@ -70,6 +70,10 @@ Provisions Kong resources with user provided Cassandra seed nodes.
 - [sa-east-1]({{ page.links.aws }}?region=sa-east-1#/stacks/new?stackName=kong-elb-pv&templateURL={{ page.links.templates.kong-pv }})
 
 ----
+### Recommended usage:
+
+	<B>Use this cloud formation as basis for your own, adjust the variables and template to better suite your needs.</B>
```
Comment (thibaultCha) [inline app/install/aws-cloudformation.md]: as *a* basis?

Comment (shashiranjan84) [inline app/install/aws-cloudformation.md]: @ahmadnassri  ^?

For me both sounds right.

Comment (ahmadnassri) [inline app/install/aws-cloudformation.md]: both are right... ask a native English speaker. :smile:

Comment (shashiranjan84) [inline app/install/aws-cloudformation.md]: Updated it. Please review.

**Pull request merged**